src/game.h

```diff
@@ -17,6 +17,7 @@
 #ifndef _GAME_H_
 #define _GAME_H_
 
+#include "addresses.h"
```

Comment:
I'd rather this was still inside a `NO_RCT2` guard (even though the header itself has one), otherwise it will cause unnecessary recompiles whenever `addresses.h` gets touched.